packages/component-library/src/LineChart/LineChart.js

```diff
@@ -0,0 +1,89 @@
+import React, { PropTypes } from 'react';
+import { css } from 'emotion';
+import {
+  VictoryScatter,
+  VictoryChart,
+  VictoryLine,
+  VictoryAxis,
+} from 'victory';
+
+import CivicVictoryTheme from '../VictoryTheme/VictoryThemeIndex';
+
+const linechartWrapper = css`
+  margin: 0 auto;
+  max-width: 900px;
+  width: 100%;
+`;
+
+const titleStyle = css`
+  font-family: 'filson-soft', sans-serif;
+  font-size: 40px;
+  font-weight: 500;
+  text-align: center;
+  margin: 40px 0 12px 0;
+`;
+
+const subtitleStyle = css`
+  display: block;
+  font-family: 'filson-soft', sans-serif;
+  font-size: 18px;
+  text-align: center;
+`;
+
+const LineChart = ({
+  title,
+  subtitle,
+  data,
+  xLabel,
+  dataKey,
+  dataValue,
+  dataKeyLabel,
+}) => (
+  <div>
+    { title ? <h3 className={titleStyle}>{title}</h3> : null}
+    { subtitle ? <span className={subtitleStyle}>{subtitle}</span> : null}
+    <div className={linechartWrapper}>
+      <VictoryChart
+        domainPadding={20}
+        animate={{ duration: 300 }}
+        theme={CivicVictoryTheme.civic}
+      >
+        <VictoryAxis
+          label={xLabel}
+          tickValues={data.map(d => d[dataKey])}
+          tickFormat={data.map(d => d[dataKeyLabel])}
+        />
+        <VictoryAxis
+          dependentAxis
+          tickValues={data.map(d => d[dataValue])}
+        />
+        <VictoryLine
+          data={data.map(d => ({ dataKey: d[dataKey], dataValue: d[dataValue] }))}
+          x="dataKey"
+          y="dataValue"
+        />
+        <VictoryScatter
+          data={data.map(d => ({ dataKey: d[dataKey], dataValue: d[dataValue] }))}
+          x="dataKey"
+          y="dataValue"
```

Comment:
If we're explicitly defining the data shape via the map functions on 61 and 66 then we can just define the keys as x/y and then omit these additional properties as the defaults for x and y are just "x" and "y". If we're going to stick with "dataKey" and "dataValue" then we should use a constant for the object keys and the props here so that anyone coming here later understands the coupling of these lines.

Reply:
Hey, we talked a bit about this issue during the meeting, and @DingoEatingFuzz posted some notes in Slack. If it's okay with you, I say we keep this for now (it's consistent with the Bar chart) then see how we can possibly standardize on a better common format as we continue to merge these charts in.